src/locales/sr/translation.json

```diff
@@ -126,6 +131,10 @@
     "thumbnailNavigation": "Сличице",
     "thumbnails": "Сличице",
     "toggleWindowSideBar": "Сајдбар",
+    "collection": "Колекција",
```

Comment:
Is this `collection` key needed here?

Reply:
Also in abecede?

Reply:
Thank you, yes please add in alphabetical direction!